### PR TITLE
migrate to gcp

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [x] In Progress
- [ ] Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-704
related: https://github.com/demisto/demisto-sdk/pull/2171

## Description
Move build servers to GCP.

### CODE REVIEW

File: Tests/test_content.py [hunk 1]
```diff
@@ -350,19 +350,19 @@ def get_server_numeric_version(ami_env, is_local_run=False):
...
-    extracted_version = re.findall(r'Demisto-(?:Circle-CI|Marketplace)-Content-AMI-[A-Za-z]*[-_](\d[._]\d)-[\d]{5}',
-                                   instances_ami_name)
-    extracted_version = [match.replace('_', '.') for match in extracted_version]
+    try:
+        server_numeric_version = re.match(
+            r'server-image-(?:ga-)?(?P<version>[a-z0-9\-]+)-(?P<build_number>\d+)-(?P<creation_date>\d{4}-\d{2}-\d{2})',
+            instances_ami_name
+        ).group('version')
+    except (AttributeError, IndexError):
+        return default_version
```
Comment (dorschw) [inline Tests/test_content.py:359]: worth a log line? (debug)

Comment (esharf) [reply Tests/test_content.py:359]: the server version is printed later

File: Tests/scripts/destroy_instances.py [hunk 2]
```diff
@@ -1,53 +1,81 @@
...
+
 def main():
     install_logging('Destroy_instances.log')
-    circle_aritfact = sys.argv[1]
-    env_file = sys.argv[2]
-    instance_role = sys.argv[3]
-    time_to_live = sys.argv[4]
-    with open(env_file, 'r') as json_file:
+    options = options_handler()
+    time_to_live = os.getenv('TIME_TO_LIVE')
```
Comment (DeanArbel) [inline Tests/scripts/destroy_instances.py:56]: Why not pass it as part of the `options`?

File: Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.py [hunk 3]
```diff
@@ -7,6 +7,8 @@
 # Disable insecure warnings
 urllib3.disable_warnings()
 
+
+
```
Comment (DeanArbel) [inline Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.py:11]: Please revert

File: Tests/test_content.py [hunk 4]
```diff
@@ -346,19 +346,19 @@ def get_server_numeric_version(ami_env, is_local_run=False):
...
 
-    if extracted_version:
-        server_numeric_version = extracted_version[0]
+    if server_numeric_version == 'master':
+        logging.info('Server version: Master')
+        return default_version
```
Comment (DeanArbel) [inline Tests/test_content.py:359]: Didn't we have a discussion about changing this to use a dedicated field for the server version?

File: pyproject.toml [hunk 5]
```diff
@@ -10,9 +10,10 @@ python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
 
 [tool.poetry.group.dev.dependencies]
-demisto_sdk = "1.15.5"
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", branch = "eli/migrate_to_gcp"}
```
Comment (DeanArbel) [inline pyproject.toml:13]: Please make sure to update this before release

File: Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.py (hunk 3 above)
Comment (yucohen) [reply Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.py:11]: Sure, added just for testing

**Pull request merged**